app/main/views/briefs.py

```diff
@@ -204,6 +206,113 @@ def update_brief_status(brief_id, action):
     return jsonify(briefs=brief.serialize()), 200
 
 
+@main.route('/briefs/<int:brief_id>/award', methods=['POST'])
+def award_brief(brief_id):
```

Comment:
Good point. Perhaps something like `award_winning_brief_response` would be more descriptive of what's happening?